zyte_api/__main__.py

```diff
@@ -71,13 +72,13 @@ def write_output(content):
                 try:
                     result = await fut
                 except Exception as e:
-                    if store_errors:
-                        write_output(e.parsed.response_body.decode())
+                    if store_errors and isinstance(e, RequestError):
+                        write_output(e.parsed.data)
 
                     if stop_on_errors:
                         raise
 
-                    logger.error(str(e))
+                    logger.exception("Exception raised during response handling")
```

Comment:
I found it useful to see the exception traceback when the exception is not `RequestError`. Although it might make things too verbose for `RequestError`. Maybe we should only log the traceback when the exception is unknown?